todolist/settings.py

```diff
@@ -61,13 +61,13 @@
 # Database
 # https://docs.djangoproject.com/en/1.7/ref/settings/#databases
 DATABASES = {
-    'default': {
-        'ENGINE': 'mysql.connector.django',
-        'NAME': 'app_db',
-        'USER': 'app_user',
-        'PASSWORD': '1234',
-        'HOST': 'mysql',  # You can use a different host if your MySQL server is on a remote machine.
-        'PORT': '',  # Leave this empty to use the default MySQL port (3306).
+    "default": {
+        "ENGINE": os.getenv("ENGINE", "mysql.connector.django"),
+        "NAME": os.getenv("NAME", "app_db"),
+        "USER": os.getenv("USER", "app_user"),
+        "PASSWORD": os.getenv("PASSWORD", "1234"),
+        "HOST": os.getenv("HOST", "mysql"),  # Значення за замовчуванням — 'mysql'
+        "PORT": os.getenv("PORT", ""),  # Якщо порт не вказано, використовується MySQL за замовчуванням (3306).
```

Comment:
The `PORT` environment variable for the database is set to an empty string. Ensure that the correct port is specified or that the default MySQL port (3306) is used.